<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidDeployQtWidget</class>
 <widget class="QWidget" name="AndroidDeployQtWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="signPackage">
     <property name="title">
      <string>Sign package</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="KeystoreLocationLabel">
          <property name="text">
           <string>Keystore:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Utils::PathChooser" name="KeystoreLocationPathChooser" native="true"/>
        </item>
        <item>
         <widget class="QPushButton" name="KeystoreCreatePushButton">
          <property name="text">
           <string>Create...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="signPackageCheckBox">
          <property name="text">
           <string>Sign package</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="signingDebugWarningIcon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../projectexplorer/projectexplorer.qrc">:/projectexplorer/images/compile_warning.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="signingDebugWarningLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Signing a debug package</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="KeystoreLocationLabel_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Certificate alias:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="certificatesAliasComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="advancedActions">
     <property name="title">
      <string>Advanced Actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QPushButton" name="cleanLibsPushButton">
        <property name="text">
         <string>Clean Temporary Libraries Directory on Device</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="installMinistroButton">
        <property name="text">
         <string>Install Ministro from APK</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="resetDefaultDevices">
        <property name="text">
         <string>Reset Default Devices</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="openPackageLocationCheckBox">
        <property name="text">
         <string>Open package location after build</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="verboseOutputCheckBox">
        <property name="text">
         <string>Verbose output</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="createAndroidManifestButton">
        <property name="text">
         <string>Create AndroidManifest.xml</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="application">
     <property name="title">
      <string>Application</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QComboBox" name="targetSDKComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="targetSDKLabel">
        <property name="text">
         <string>Android build SDK:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="inputFileComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="inputFileLabel">
        <property name="text">
         <string>Input file for androiddeployqt:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="oldFilesWarningIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../projectexplorer/projectexplorer.qrc">:/projectexplorer/images/compile_warning.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="oldFilesWarningLabel">
       <property name="text">
        <string>Qt no longer uses the folder &quot;android&quot; in the project's source directory.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="qtDeployment">
     <property name="title">
      <string>Qt Deployment</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="ministroOption">
        <property name="toolTip">
         <string>Uses the external Ministro application to download and maintain Qt libraries.</string>
        </property>
        <property name="text">
         <string>Use Ministro service to install Qt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="temporaryQtOption">
        <property name="toolTip">
         <string>Pushes local Qt libraries to device. You must have Qt libraries compiled for that platform.
The APK will not be usable on any other device.</string>
        </property>
        <property name="text">
         <string>Deploy local Qt libraries to temporary directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bundleQtOption">
        <property name="toolTip">
         <string>Creates a standalone APK.</string>
        </property>
        <property name="text">
         <string>Bundle Qt libraries in APK</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="additionalLibrariesGroupBox">
     <property name="title">
      <string>Additional Libraries</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="androidExtraLibsLayout">
        <item>
         <widget class="QListView" name="androidExtraLibsListView">
          <property name="toolTip">
           <string>List of extra libraries to include in Android package and load on startup.</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="androidExtraLibsButtonLayout">
          <item>
           <widget class="QToolButton" name="addAndroidExtraLibButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select library to include in package.</string>
            </property>
            <property name="text">
             <string>Add...</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeAndroidExtraLibButton">
            <property name="toolTip">
             <string>Remove currently selected library from list.</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../projectexplorer/projectexplorer.qrc"/>
 </resources>
 <connections/>
</ui>
